src/components/greeting.js

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchGreetings } from '../redux/greetingSlice';
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+  const { greeting, loading } = useSelector((store) => store.greeting);
```

Comment:
## [OPTIONAL]
- [ ] I would suggest you to add also some error handling so that the user will know what is going on if the request failed

Reply:
@andersonlebon Done